src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -59,53 +60,61 @@ export class JailAddComponent implements OnInit {
       type: 'checkbox',
       name: 'dhcp',
       placeholder: T('DHCP autoconfigure IPv4'),
-      tooltip: T('Check to start the jail with the Dynamic Host\
- Configuration Protocol enabled. <b>VirtIO</b> and <b>Berkeley Packet\
- Filter</b> must also be enabled.'),
+      tooltip: T('Set to start the jail with the Dynamic Host \
+                 Configuration Protocol enabled. <b>VirtIO</b> and \
+                 <b>Berkeley Packet Filter</b> must also be enabled.'),
     },
     {
       type: 'checkbox',
       name: 'vnet',
       placeholder: T('VirtIO Virtual Networking'),
```

Comment:
I believe this placeholder needs to change to "VNET" (confirm with Brandon)

Reply:
Confirmed with Brandon. Updated to VNET and linked to the correct VNET(9) man page